libvast/src/format/json.cpp

```diff
@@ -145,6 +145,43 @@ struct convert {
   }
 };
 
+struct relaxed_convert : strict_convert {
+  using strict_convert::operator();
+
+  caf::expected<data> operator()(json::string str, const bool_type&) const {
+    if (bool res; parsers::json_boolean(str, res))
```

Comment:
```suggestion
    if (bool x; parsers::json_boolean(str, res))
```

I think just using `x` would make things a bit more readable. The `res` things feel like a sub-optimal compromise between `result` and a less verbose token.